<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Steuerung</class>
 <widget class="QWidget" name="Steuerung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>0</y>
     <width>352</width>
     <height>563</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="setLayout">
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Iteration Interval (in msec)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="iterationInterval">
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Alpha</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="alpha">
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Beta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="beta">
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>K0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="k0">
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Max. No. Iterations</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="maxIter">
      <property name="maximum">
       <number>2000000000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>CV ratio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="cvRatio">
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="radius">
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>City Value Interval</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="CityLayout">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fromCity"/>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="toCity"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Node Value Interval</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="NodeLayout">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fromNode"/>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="toNode"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Eta Destination</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="etaDestination">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="TSPLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
